### PR TITLE
chore: redesign landing page

### PR DESCRIPTION
**Pull request by rdubrock**
Screenshot 2024-04-12 at 12 08 31
Screenshot 2024-04-15 at 12 40 07
Screenshot 2024-04-15 at 12 39 59

### CODE REVIEW

**Comment by roobre** [comment]
Can't say much about the code, but I like the new look! 🎉

**Review by ckbedwell — CHANGES_REQUESTED** [review]
Few comments all very small. General ones:

1. Putting on my accessibility police hat, I don't like any of the min / max width properties on a few of the elements as they don't work great with responsiveness. You can definitely achieve what you want whilst making them respond when resolutions get smaller 😄 

2. Can we add some margin / padding to the bottom of the page? The code snippets has a bit but the images don't have, so that felt a bit weird as I was scrolling.

Otherwise, I think this is great. Good job! 💪

File: src/components/WelcomeTabs/WelcomeTabContent.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        </>
+      );
+    case WelcomeTab.PrivateProbes:
+      return (
+        <>
+          <p>
+            In addition to the locations we provide out of the box, you can set up your own probes to run inside your
+            network or from a location of your choosing
+          </p>
+          <img src={config.theme2.isDark ? privateProbeDark : privateProbeLight} className={styles.screenshot} />
```
Comment (ckbedwell) [inline src/components/WelcomeTabs/WelcomeTabContent.tsx:61]: 🚨 Forgot to leave my other accessibility police comment: Alt text (for all the images), please 😄

File: src/components/CodeSnippet/CodeSnippet.tsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    () => snippet && Prism.highlight(formatter(snippet), Prism.languages[langSyntax] as Grammar, langSyntax),
+    [formatter, snippet, langSyntax]
+  );
+
+  return (
+    <SnippetWindow
+      titleContent={
+        tabs.length > 0 && (
+          <TabsBar className={styles.tabsBar}>
+            {tabs.flatMap((tab) => {
```
[the remark this answers is not in the document]
Comment (rdubrock) [reply src/components/CodeSnippet/CodeSnippet.tsx:106]: Nope! This code changed a bunch and I forgot to update

**Review by ckbedwell — APPROVED** [review]
LGTM

**Pull request merged**